packages/tpetra/core/src/Tpetra_Import_decl.hpp

```diff
@@ -314,6 +314,24 @@ namespace Tpetra {
     Import<LocalOrdinal,GlobalOrdinal,Node>&
     operator= (const Import<LocalOrdinal,GlobalOrdinal,Node>& Source);
 
+    /// \brief Build the target map for the union of two Import objects
+    ///
+    /// This procedure assumes that you have already preprocessed
+    /// all the GIDs to identify remotes.  This involves using the two
+    /// input Import objects, namely converting their permute and remote
+    /// LID lists to GIDs.  You'll also need to extract PIDs from the
+    /// Imports in order to build pids_map.
```

Comment:
Arghh, forgot to clean up the documentation before pushing.  This should be changed to reflect the actual behavior now (and not notes to myself about what I need to do)